data/data/azure/variables-azure.tf

```diff
@@ -106,3 +106,8 @@ variable "azure_compute_subnet" {
   type        = string
   description = "The name of the subnet for worker nodes, either existing or to be created"
 }
+
+variable "azure_publish" {
+  type        = string
```

Comment:
I [put this in my GCP review](https://github.com/openshift/installer/pull/2522#discussion_r336047938) but I think we should pass this as a bool to TF corresponding to public/private, in this case I guess it would be `private`.

Reply:
Agreed. I'll fix this shortly.